wiki/Client/Beatmap_editor/Menu/de.md

```diff
@@ -245,5 +135,14 @@ Interessante Dinge.
 
 | Name | Beschreibung |
 | :-- | :-- |
-| Ingame-Hilfe anzeigen (`H`) | Öffnet im Editor ein Fenster mit hilfreichen Informationen. |
-| FAQ anzeigen | Verlinkt nach [Beatmapping](/wiki/Beatmapping) |
+| Ingame-Hilfe anzeigen (`H`) | Öffnet ein Fenster mit Tastenbelegungen und hilfreichen Informationen.[^help-note] |
```

Comment:
In an [earlier version](https://github.com/ppy/osu-wiki/commit/0563cac42477b446e16d1aeaceb3d0edb9d55f16#diff-d31ff5ffd1ddb668dc9c25c1170a031516478292e8934502a6c8eaa678498036R255), this button was still present.

I guess we can remove it because the button has lost its function in the meantime.